redbeams-api/src/main/java/com/sequenceiq/redbeams/api/endpoint/v4/database/base/DatabaseV4Base.java

```diff
@@ -0,0 +1,94 @@
+package com.sequenceiq.redbeams.api.endpoint.v4.database.base;
+
+import static com.sequenceiq.redbeams.doc.ModelDescriptions.DESCRIPTION;
+import static com.sequenceiq.redbeams.doc.ModelDescriptions.Database;
+import static com.sequenceiq.redbeams.doc.ModelDescriptions.ENVIRONMENT_ID;
+
+import java.io.Serializable;
+
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Pattern;
+import javax.validation.constraints.Size;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.sequenceiq.cloudbreak.validation.ValidRDSConfigJson;
+
+import io.swagger.annotations.ApiModelProperty;
+
+@ValidRDSConfigJson
+@JsonIgnoreProperties(ignoreUnknown = true)
+public abstract class DatabaseV4Base implements Serializable {
+
+    @NotNull
+    @Size(max = 100, min = 5, message = "The length of the database config's name has to be in range of 5 to 100")
```

Comment:
nit: Replace "has to be in range of 5 to 100" with "must be between 5 and 100".

Reply:
Feel free to look at DatabaseServerV4Base to steal verbiage.

Reply:
done, thanks.